### ISSUE TITLE
User filter when searching for logs prevents Automated Process logs from showing up

### ISSUE THREAD

**Issue by robertoandrade** [motiko/sfdc-debug-logs#10]
Even though I have `Automated Process` on the list of traced entities, since it doesn't have a hyperlink to the user it doesn't include it on the filtered logs.

I wonder if the user filter is really necessary since the default implementation from SF doesn't filter the listing of logs other than by the search criteria.

**Comment by motiko** [comment]
As a developer or administrator on big orgs i would rather see my own logs by default.
Would be happy to approve a PR that makes this filter optional but would love to keep at as default.
Let me know if you need any help with building from source.

**Comment by robertoandrade** [comment]
Makes sense, happy to submit a PR, any instructions on how to go about building the extension?

**Comment by motiko** [comment]
from react folder 
```
npm i
npm run build
```
then load chrome/app as an extension in chrome
not sure where you need to change to not filter by user, have to dig into SF API docs,
but my guess would be some where around
https://github.com/motiko/sfdc-debug-logs/blob/master/react/src/api/tooling.js

**Comment by robertoandrade** [comment]
Ok, thanks. I noticed this also affects the improved Debug Logs view (not only the separate UI you provide). I assume the code that extends the SF classic UI with the extra buttons, rows, limit, etc. is somewhere else?

**Comment by robertoandrade** [comment]
ie: 
![image](https://user-images.githubusercontent.com/318969/75720919-1dae8000-5cb6-11ea-8fab-92faa843718c.png)

**Comment by robertoandrade** [comment]
guessing [here](https://github.com/motiko/sfdc-debug-logs/blob/962f78fe791c063426c720d42dc4d34c5c19affe/chrome/app/js/sfdc.debug.btn.js#L280). But how does the code in `/react/src` relate to the ones in `/chrome/app` and `/firefox/app`? I thought all the code in there (or most of it) was generated from the `/react/src` but it looks like it's actually only the `app.js` that gets added in there. The code for the SF classic UI enhancements is duplicated between the two browser extension directories?

**Comment by motiko** [comment]
for the old view just [this file](https://github.com/motiko/sfdc-debug-logs/blob/master/chrome/app/js/sfdc.debug.btn.js) need changing and no need for build step. so you can load `chrome/app` directly

**Comment by motiko** [comment]
> guessing [here](https://github.com/motiko/sfdc-debug-logs/blob/962f78fe791c063426c720d42dc4d34c5c19affe/chrome/app/js/sfdc.debug.btn.js#L280). But how does the code in `/react/src` relate to the ones in `/chrome/app` and `/firefox/app`? I thought all the code in there (or most of it) was generated from the `/react/src` but it looks like it's actually only the `app.js` that gets added in there. The code for the SF classic UI enhancements is duplicated between the two browser extension directories?

Yes it is mostly duplicated..

**Comment by motiko** [comment]
> ie:
> ![image](https://user-images.githubusercontent.com/318969/75720919-1dae8000-5cb6-11ea-8fab-92faa843718c.png)

thats the place you need imo

**Comment by robertoandrade** [comment]
That's exactly where I was pointing earlier. My point is, it seems like that code is duplicated between the `/chrome/app` and `/firefox/app` directories, so I was wondering if the place you were pointing at earlier (`/react/src`) had anything to do with those copies, ie: the source resided there and after compiling it'd be placed in each browser's app folder, but couldn't find anything that ressembled that in the `/react/src` folder.

**Comment by motiko** [comment]
The code for the  SF classic UI view is duplicated. 
**Additional** code for the tab is populated from react folder with `npm build` .
If your change is SF classic UI you only need to change code in `chrome/app`

**Comment by robertoandrade** [comment]
Ok, when creating the PR shouldn't it then copy the changed code in
`chrome/app` to `firefox/app`?

**Comment by motiko** [comment]
It is not a must, but if you can do same changes for firefox too would be nice. 
In that case make sure to test it works on FF as well. 🙏

**Comment by motiko** [comment]
@robertoandrade you need any help with that?
you can actually leave firefox part out. just submit the code in the PR and i will take care of the build.
